src/main/java/ca/corefacility/bioinformatics/irida/service/impl/StaticContextInitializer.java

```diff
@@ -0,0 +1,28 @@
+package ca.corefacility.bioinformatics.irida.service.impl;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import ca.corefacility.bioinformatics.irida.repositories.filesystem.IridaFileStorageUtility;
+import ca.corefacility.bioinformatics.irida.util.IridaFiles;
+
+/**
+ * Allows for setting the iridaFileStorageUtility for static classes
+ */
+
+@Component
+public class StaticContextInitializer {
+
+	@Autowired
+	private IridaFileStorageUtility iridaFileStorageUtility;
+
+	/**
+	 * Sets the iridaFileStorageUtility in the IridaFiles static class
+	 */
+	@PostConstruct
+	public void init() {
+		IridaFiles.setIridaFileStorageUtility(this.iridaFileStorageUtility);
```

Comment:
This works, but I think we could get rid of this file.  Since you're configuring `IridaFileStorageUtility` in `IridaApiServicesConfig`, I think you could call `IridaFiles.setIridaFileStorageUtility` right after you create it in that file.  Maybe try it out.

Reply:
It works with IridaFiles.setiridafilestorageutility being set in the IridaApiServicesConfig. Removed this file in 97e0014e1c5aa5584ad4b4f07bf67f3d54fbaf70